### PR TITLE
Readme improved

### PR DESCRIPTION
**Pull request by puneith**
@nkashy1 @rhaertel80 @dizcology @sgreenberg @jingjinglong

### CODE REVIEW

**Comment by puneith** [comment]
@nkashy1 @rhaertel80 ping on this!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+We are happy to see your enthusiasm. This table lists the criteria and how important is it to follow when you make your contributions. 
+
+<br/>
+
+| Criteria        | Core         | Contrib  |
+| ------------- |:-------------:| -----:|
+| Include requirements.txt for dependencies     | Hard | Hard |
+| Include unit tests     | Hard | Soft |
+| Include integration tests     | Hard | Soft|
+| Include core TF API aka no contrib   | Hard | Soft |
```
Comment (rhaertel80) [inline CONTRIBUTING.md]: I don't undersatnd this one. Can you explain this a bit more?

Comment (puneith) [reply CONTRIBUTING.md]: The idea is to list which criteria is hard (required) and which is soft (optional). Does that change in wording help or the way it reads its unclear what this table really is?

Comment (elmer-garduno) [reply CONTRIBUTING.md]: Also not clear exactly what Core or Contrib refers to, after a few minutes I figured out it refers to the samples themselves but my first thought was it referred to TF core or TF contrib.

Comment (puneith) [reply CONTRIBUTING.md]: @elmer-garduno @rhaertel80 how about now?

**Pull request merged**